xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxBorrowingService.java

```diff
@@ -0,0 +1,124 @@
+package org.knowm.xchange.ftx.service;
+
+import org.knowm.xchange.ftx.FtxExchange;
+import org.knowm.xchange.ftx.dto.account.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class FtxBorrowingService extends FtxBaseService  {
+
+  public FtxBorrowingService(FtxExchange exchange) {
+    super(exchange);
+  }
+
+  public List<FtxBorrowingsDto> histories(String subaccount) {
+    try {
+      return ftx.getBorrowHistory(
+          exchange.getExchangeSpecification().getApiKey(),
+          exchange.getNonceFactory().createValue(),
+          signatureCreator,
+          subaccount
+      ).getResult();
+    } catch (IOException e) {
+      throw new FtxLendingService.FtxLendingServiceException("Can't get lending infos subAccount: " + subaccount, e);
+    }
+  }
+
+  public List<FtxBorrowingsDto> histories(String subaccount, List<String> coins) {
+    Objects.requireNonNull(coins);
+    return histories(subaccount).stream()
+        .filter(lendingHistory -> coins.contains(lendingHistory.getCoin()))
+        .collect(Collectors.toList());
+  }
+
+  public List<FtxBorrowingsDto> histories(String subaccount, String... coins) {
+    Objects.requireNonNull(coins);
+    return histories(subaccount, Arrays.asList(coins));
+  }
+
+  public FtxBorrowingsDto history(String subaccount, String coin) {
+    Objects.requireNonNull(coin);
+    return histories(subaccount).stream()
+        .filter(lendingHistory -> lendingHistory.getCoin().equalsIgnoreCase(coin))
+        .findFirst()
+        .orElse(null);
+  }
+
+  public List<FtxBorrowingInfoDto> infos(String subaccount) {
+    try {
+      return ftx.getBorrowingInfos(
+          exchange.getExchangeSpecification().getApiKey(),
+          exchange.getNonceFactory().createValue(),
+          signatureCreator,
+          subaccount
+      ).getResult();
+    } catch (IOException e) {
+      throw new FtxLendingService.FtxLendingServiceException("Can't get lending infos subAccount: " + subaccount, e);
+    }
+  }
+
+  public List<FtxBorrowingInfoDto> infos(String subaccount, List<String> coins) {
```

Comment:
do we need this and the signature below, they are the same thing, I would keep it simple and just provide either array or list signature.